build/deploy/examples/minimum/main.jsonnet

```diff
@@ -19,9 +19,13 @@ local metadata = metadataBase {
     storageClass: 'VAR_CRDB_STORAGE_CLASS',
   },
   gateway+: {
-    ipName: 'VAR_INGRESS_NAME',
     image: 'VAR_DOCKER_IMAGE_NAME',
     hostname: 'VAR_APP_HOSTNAME',
+    ingress: 'none', // <-- This string value is VAR_INGRESS
```

Comment:
I imagine there will be other things we would condition on cloud type; could we set the cloud type only once somewhere and then use that value here (and other places later)?